docs/user-guide/image-plot.ipynb

```diff
@@ -186,6 +186,30 @@
     "da.masks['negative'] = da.data < sc.scalar(0, unit='m/s')\n",
     "da.plot()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "234f96fc-b05c-42e9-8443-67c8c8865560",
+   "metadata": {},
+   "source": [
+    "## Using a non-dimension coordinate\n",
+    "\n",
+    "By default, Plopp will use the dimension coordinates to set the horizontal position of the points ad the axis labels.\n",
+    "To use a non-dimension coordinate instead, use the `coords` argument."
```

Comment:
I am not sure every reader will be familiar with (non)-dimension coords. Have you considered writing it differently, e.g.,

"... for a dimension of name "abc", plopp will use the corresponding coordinate of name "abc". To use a different coordinate, use the `coords` argument.". Should also change the subsection title accordinly.

What if there is no "abc" coord? It is just using an `arange`, right? Maybe worth mentioning as well.

Reply:
done